### PR TITLE
pkg: append PL to lightning

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

NO COMMENT :otter: 

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda @carmocca @akihironitta

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: .azure/ipu-tests.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ pr:
       - ".actions/**"
       - ".azure/ipu-tests.yml"
       - "requirements/fabric/**"
+      - "src/lightning/*"
```
Comment (carmocca) [inline .azure/ipu-tests.yml:21]: Can we be explicit with what we are adding here?

Also these changes need to be reflected in the checkgroup config too

Comment (Borda) [reply .azure/ipu-tests.yml]: added :)

File: .azure/app-cloud-store.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@ pr:
   paths:
     include:
       - ".azure/app-cloud-store.yml"
+      - "src/lightning/*.py"
```
Comment (carmocca) [inline .azure/app-cloud-store.yml:24]: I think you missed the checkgroup change associated to this change

File: src/lightning/__setup__.py [hunk 3]
```diff
@@ -83,9 +87,7 @@ def _setup_args() -> Dict[str, Any]:
             ],
         },
         setup_requires=[],
-        install_requires=_ASSISTANT.load_requirements(
-            _PATH_REQUIREMENTS, unfreeze="none" if _FREEZE_REQUIREMENTS else "major"
-        ),
+        install_requires=base_requires,
```
Comment (carmocca) [inline src/lightning/__setup__.py:90]: ultra nit: make both names match?

**Review by carmocca — APPROVED** [review]

File: .azure/app-cloud-e2e.yml [hunk 4]
```diff
@@ -29,6 +29,7 @@ pr:
     include:
       - ".actions/**"
       - ".azure/app-cloud-e2e.yml"
+      - "src/lightning/*.py"
```
Comment (carmocca) [inline .azure/app-cloud-e2e.yml:32]: @Borda This is not working correctly. PL PRs are now running app and store tests. See https://github.com/Lightning-AI/lightning/pull/16926 for example

**Pull request merged**